import logging as log

log.error("Esto es un log  de prueba")